backend/src/routes/units.ts

```diff
@@ -16,4 +16,6 @@ router.get("/:id", UnitController.getUnitHandler);
 
 router.post("/", validateWith(createUnitValidators), UnitController.createUnitsHandler);
 
+router.put("/:id/approve", UnitController.approveUnitHandler);
```

Comment:
Can we gate this route using the `requireHousingLocator` middleware?

Reply:
Updated